Automattic/jetpack
# Business Hours:  Add color, typography, spacing & alignment design tools.

### Changes proposed in this Pull Request:

Add design tools for Business Hours block.

![2022-01-27_13-40-08](https://user-images.githubusercontent.com/1287077/151353017-1ff42006-edc9-4a7a-957f-be2d568f04ea.png)

![2022-01-27_13-41-11](https://user-images.githubusercontent.com/1287077/151353028-8d0d3e31-c856-4f10-a93b-523c5f135be3.png)

#### Jetpack product discussion
This PR is a part of the Expanding Jetpack Block design tooling for Full Site Editing project: pdDOJh-1P-p2 & pdDOJh-6-p2

#### Does this pull request change what data or activity we track or use?
No, it does not.

#### Testing instructions:

1. Add "Business Hours" block on the editor.
2. On the **Block Inspector** sidebar, you should see:
    - Color (Text, Background with Gradient option in the color picker)
    - Typography (Size & Line Height option)
    - Dimensions (Padding & Margin option)
3. On the **Block Toolbar,** you should see:
    - Align (Full & Wide align option)
4. Setting any of the options above should render on the editor and when previewed/published.

Note: Different themes support different style attributes. Some theme suggestions to test on: Quadrat, Geologist, TT1 (Blocks), Seedlet (Blocks).
